worksheet_b/README.MD

```diff
@@ -0,0 +1,53 @@
+
+
+Print Welcome to
+Terminal 4
+
+Print Password
+Required
+
+Secret Word = Water
+
+Player Strikes Left
+= 5
+
+Computer generates
+the 11 words in a random order
+
+                Computer words = ("Bales,
+Towel, Water, Artsy, Fairy, Chair, Wacky, Voice, Dough,            Grass, Knee, Water.")
+
+If Player word is
+not equal to secret word player loses strike 
+
+                if Letters = same position as
+Secret Word display similiarity 
+
+                If player word != Secret word
+minus 1 from player strikes 
+
+Player guesses a
+word 
+
+                if Player word = Secret word
+player wins 
+
+                if Player word != Secret Word
+
+loop
```

Comment:
Where does the program loop back to? It would be clearer to structure this as e.g. a while loop.